echo/frontend/src/routes/Debug.tsx

```
@@ -0,0 +1,836 @@
+        originalRow && typeof originalRow.id === "string"
+          ? originalRow.id
+          : `idx_${index}`;
+
+      // For parent-child relationships, create a path-like ID structure
+      if (parent) {
+        // This ensures we have a unique hierarchy path for each row
+        return `${parent.id}_${originalId}`;
+      }
+
+      // Add timestamp if available to further ensure uniqueness
+      if (originalRow && originalRow.timestamp) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)